charts/kargo/templates/api/configmap.yaml

```
@@ -18,6 +18,9 @@ data:
   TLS_CERT_PATH: /etc/kargo/tls.crt
   TLS_KEY_PATH: /etc/kargo/tls.key
   {{- end }}
+  {{- if .Values.api.enableSecretManagement }}
+  ENABLE_SECRET_MANAGEMENT: "true"
```

```suggestion
  SECRET_MANAGEMENT_ENABLED: "true"
```

would match the style of `PERMISSIVE_CORS_POLICY_ENABLED` and `ADMIN_ACCOUNT_ENABLED` below.